qubesmanager/log_dialog.py

```diff
@@ -56,7 +58,7 @@ def __init_log_text__(self):
         if log.tell() > LOG_DISPLAY_SIZE:
             self.displayed_text = self.tr(
                 "(Showing only last %d bytes of file)\n") % LOG_DISPLAY_SIZE
-            log.seek(-LOG_DISPLAY_SIZE, os.SEEK_END)
+            log.seek(log.tell()-LOG_DISPLAY_SIZE, os.SEEK_SET)
```

Comment:
What was wrong with the old version?

Reply:
Due to changes in file.seek() behavior in Python3, it didn't work at all - offset must be 0 with SEEK_END in case of text files. Reference: https://docs.python.org/3.2/library/io.html#io.TextIOBase.seek